sites/upsun/src/increase-observability/application-metrics/cron-metrics.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Monitor Cron job executions
+description: Learn how to visualize and interpret your Cron job runs directly in the console.
+weight: -5
+keywords:
+  - "cron"
+  - "jobs"
+  - "monitor"
+  - "logs"
+  - "console"
+---
+
+You can view your [Cron job](/create-apps/app-reference/single-runtime-image.html#crons) executions directly from your project’s Services > Apps or Services > Workers pages. This visualization helps you quickly understand how scheduled jobs are performing and spot unusual patterns without manually checking logs.
+
+Each execution is represented visually, letting you monitor performance trends and failures at a glance.
+
+This makes it easier to:
+
+- Verify that scheduled jobs are running on time.
+- Identify failed or unusually long-running jobs.
+- Drill into logs only when something looks wrong.
+
+## How to view Cron job executions
+
+1. Open your project in the Console.  
+2. Navigate to:
+   - **Services > Apps** or  
```

Comment:
Apps and Workers are both under **Apps**

<img width="391" height="367" alt="image" src="https://github.com/user-attachments/assets/219dde53-a652-441e-abc7-22862c93db4f" />

Reply:
so 
Services --> Apps & Services --> Apps --> app
or
Services --> Apps & Services --> Apps --> Workers